### ISSUE TITLE
Animation Issues

### ISSUE THREAD

**Issue by Byterset** [lambertjamesd/n64brew2022#1]
Hi!

The game is able to build and is completely playable but skeletal animations are not working at all. For exporting the fbx Blender 2.93.12 is used and the skelatool version is the one included in this repo. Any obvious things I am doing wrong?
Maybe worth noting that the make command will run into errors during compilation because conveyor.h, table.h and clips.h are not beeing built automatically and have to be built seperately with make but maybe that is just a different issue

**Comment by lambertjamesd** [comment]
I can't say for sure what is wrong. Do the animations simply not play and the objects are stuck in the same pose the entire time?

The compilation errors are probably an issue with the Makefile but probably not related the the animations not working.

**Comment by Byterset** [comment]
Yeah they don't play so every animated object is stuck in the initial positon.
I was wondering if it could have something to do with how the models are exported to fbx, maybe the blender version is relevant? What version did you use during development?
